src/AbstractGenerator.h

```diff
@@ -177,6 +177,21 @@ class AbstractGenerator {
      */
     virtual bool emit_cpp_stub(const std::string &stub_file_path) = 0;
 
+    /** Emit a Serialized Halide Pipeline (.hlpipe) file to the given path. Not all Generators support this.
+     *
+     * If you call this method, you should not call any other AbstractGenerator methods
+     * on this instance, before or after this call.
+     *
+     * If the Generator is capable of emitting a hlpipe, do so and return true. (Errors
```

Comment:
"an hlpipe" instead of "a hlpipe"? (How are you pronouncing it?)

Reply:
Sure ... I've been calling it an "HL" Pipe, so "an" seems more appropriate here.